chapter1/README.md

```diff
@@ -0,0 +1,403 @@
+urlFragment: "ms-identity-javascript-angular-spa-dotnetcore-webapi-roles-groups/Chapter1"
+---
+
+# An Angular Single-page Application (SPA) that authenticates users with Azure AD and calls a protected ASP.NET Core Web API using Azure AD App Roles
+
+## Overview
+
+This sample demonstrates a cross-platform application suite involving an Angular SPA (*TodoListSPA*) calling an ASP.NET Core Web API (*TodoListAPI*) secured with Azure Active Directory. It also implements **role-based access control** by using Azure AD App Roles: in the sample, a dashboard component allowing to see the tasks assigned to any user is only accessible by users under the **TenantAdmin** role.
+
+### Scenario
+
+- TodoListSPA uses [MSAL.js](https://github.com/AzureAD/microsoft-authentication-library-for-js) and [MSAL-Angular](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-angular) to authenticate a user.
+- The app then obtains an [access token](https://docs.microsoft.com/azure/active-directory/develop/access-tokens) from Azure Active Directory (Azure AD) on behalf of the authenticated user for the TodoListAPI.
+- The access token is then used by the TodoListAPI to authorize the user.
+- TodoListAPI uses [MSAL.NET](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet) and [Microsoft.Identity.Web](https://github.com/AzureAD/microsoft-identity-web) to protect its endpoint and accept authorized calls.
+
+![Topology](../ReadmeFiles/topology.png)
+
+## Contents
+
+| File/folder       | Description                                |
+|-------------------|--------------------------------------------|
+| `AppCreationScripts` | Contains Powershell scripts to automate app registrations. |
+| `TodoListAPI`     | Source code of the TodoList API.           |
+| `TodoListSPA`     | Source code of the TodoList client SPA.    |
+| `CHANGELOG.md`    | List of changes to the sample.             |
+| `CONTRIBUTING.md` | Guidelines for contributing to the sample. |
+| `LICENSE`         | The license for the sample.                |
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en/download/) must be installed to run this sample.
+- [Dotnet Core SDK](https://dotnet.microsoft.com/download) must be installed to run this sample.
+- An Azure Active Directory (Azure AD) tenant.
+- At least **two** user accounts in your Azure AD tenant.
+- A modern Browser. This sample uses **ES6** conventions and will not run on **Internet Explorer**.
+- We recommend [VS Code](https://code.visualstudio.com/download) for running and debugging this cross-platform application.
+
+## Setup
+
+Using a command line interface such as VS Code integrated terminal, follow the steps below:
+
+### Step 1. Install .NET Core API dependencies
+
+```console
+   cd TodoListAPI
+   dotnet restore
+```
+
+### Step 2. Trust development certificates
+
+```console
+   dotnet dev-certs https --clean
+   dotnet dev-certs https --trust
+```
```

Comment:
```suggestion
On OSX or Windows
```console
   dotnet dev-certs https --clean
   dotnet dev-certs https --trust
```
On Linux
```console
   dotnet dev-certs https --clean
   dotnet dev-certs https
```
```

Reply:
this should be the same on all OSs, no?